user.py

```diff
@@ -22,12 +23,12 @@ def get_id_user(id):
         user_obj = User(id=user['id'],
                         name=user['name'],
                         email=user['email'],
-                        profile_pic= user['profile_pic'],
+                        profile_pic=user['profile_pic'],
                         food_consumptions=user['food_consumptions'])
         return user_obj
 
     @staticmethod
-    def get(db,email):
+    def get(db, email):
```

Comment:
Using any specific code styling tool?

Reply:
No tool. It's just pep8 standards.

Reply:
I was asking if you changed everything manually? or used a style guide/sheet/tool for that?

Reply:
I am following pep8 standards. You can find this at (https://www.python.org/dev/peps/pep-0008/)

Reply:
I am using vscode